### ISSUE TITLE
Enhancment Request : display tiles count 

### ISSUE THREAD

**Issue by EmTee70** [gnunn1/tilix#1116]
First of all, I like Tilix! 
It was the number one applications not available for our old Ubuntu 14.04. Server that made me make the update yesterday!

I would like if somewhere in the title bar or beside the arrows to return to the tiled view "-><-"
the total number of sub windows would be displayed.

Sometimes I have two Tilix's Windows open one for a longer time and another for a short look, when doing something different between it is sometimes difficult to see which one is which. 
OK, I might rename the windows... that would help...
But the default title/name might include the number of sub-windows... :-)

Regards mt.

**Comment by gnunn1** [comment]
Just to make sure I understand, you want a title variable that represents the number of the current window? So if you had three windows there would be a variable that displays 1, 2 or 3?

**Comment by EmTee70** [comment]
No, I was meaning not the number of the current window, but this **would be be a good idea, too**!

I was meaning to display the count of sub windows = tiles inside a window, when you are in the enlarged view (after a klick to '+'),  beside the -><- arrows in the upper right corner.

Another way to automatically  make a distinction between two different windows by title, might be to set the title to the first line of input or output to of the window? 

To summarize it: make an other default title -even using a (different) random symbol like:★,  ☘ would be ok. ☺

**Comment by phw** [comment]
I think translated to Tilix terms this means having a variable the shows the total number of terminals inside a session (e.g. terminalCount, similar to how we have sessionCount variable now). Use case is to see the number of terminals in case a terminal is maximized. Core problem here probably is that a maximized terminal is hard to distinguish from a session with only one terminal).

**Comment by EmTee70** [comment]
>  Core problem here probably is that a maximized terminal is hard to distinguish from a session with only one terminal

(maximized too!)
Yes!

**Comment by EmTee70** [comment]
Cool!
More general question, when will it be 'normally' available if I don't use your latest version from github?

**Comment by gnunn1** [comment]
In general when I feel like doing a release, there's no schedule. However in this case I do want to push out a new release to fix a couple of bugs so hopefully this weekend.

**Comment by EmTee70** [comment]
Thank you!!! https://gnunn1.github.io/tilix-web/manual/title/ now needs a little update, too.
